src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs

```diff
@@ -430,7 +431,14 @@ public string AddReference(string id, string commentId)
             IReadOnlyList<string> typeGenericParameters = null,
             IReadOnlyList<string> methodGenericParameters = null)
         {
-            return _generator.AddSpecReference(symbol, typeGenericParameters, methodGenericParameters, _references, this);
+            try
+            {
+                return _generator.AddSpecReference(symbol, typeGenericParameters, methodGenericParameters, _references, this);
+            }
+            catch (Exception ex)
+            {
+                throw new DocfxException($"Unable to generate spec reference for {VisitorHelper.GetCommentId(symbol)}", ex);
```

Comment:
it is contained in inner exception.

---
In reply to: [77291836](https://github.com/dotnet/docfx/pull/665#discussion_r77291836) [](ancestors = 77291836)